src/Og.php

```diff
@@ -17,6 +17,137 @@
 class Og {
 
   /**
+   * Static cache for groups per entity.
+   *
+   * @var array
+   */
+  protected static $entityGroupCache = [];
+
+  /**
+   * Create an organic groups field in a bundle.
+   *
+   * @param $field_name
+   *   The field name.
+   * @param $entity_type
+   *   The entity type.
+   * @param $bundle
+   *   The bundle name.
+   */
+  public static function createField($field_name, $entity_type, $bundle) {
```

Comment:
Shall we follow up this? or add here? I don't mind too much, but this was just moved from the OG controller class.